### ISSUE TITLE
[BUG] npm based install not working

### ISSUE THREAD

**Issue by mswoff** [FlowiseAI/Flowise#605]
**Describe the bug**
Following the instructions under quickstart doesn't work.

**To Reproduce**

update to latest node
update to latest npm
```
npm install -g flowise
npx flowise start
```
Yields 
```
2023-07-23 16:06:45 [ERROR]: ❌ [server]: Error during Data Source initialization: AbortController is not defined
```
Then, when I try to do a flow (e.g. Simple LLM Chain), I enter in my key, save, start a chat, then type anything, I see
```
2023-07-23 16:08:31 [ERROR]: [server]: Error: Cannot read property 'activeChatflows' of undefined
TypeError: Cannot read property 'activeChatflows' of undefined
    at isFlowReusable (/Users/m/.nvm/versions/node/v14.21.3/lib/node_modules/flowise/dist/index.js:643:80)
    at App.processPrediction (/Users/m/.nvm/versions/node/v14.21.3/lib/node_modules/flowise/dist/index.js:676:21)
    at async /Users/m/.nvm/versions/node/v14.21.3/lib/node_modules/flowise/dist/index.js:427:13
2023-07-23 16:08:31 [INFO]: ⬆️ POST /api/v1/chatmessage/3ad47d6c-5142-4764-9f7f-5a0f8d01f4cf
```

**Setup**
-   Installation `npx flowise start`
-   Flowise Version: didn't specify
-   OS: macOS
-   Browser: hrome

**Comment by chungyau97** [comment]
Hi @mswoff,

Could you try using the [Node.js](https://nodejs.org/en) version of `18.17.0 LTS`
Highly recommend installing [NVM](https://github.com/nvm-sh/nvm)

**Comment by twilwa** [comment]
Hi, I'll piggyback off this as I'm experiencing a somewhat similar issue. My environment is Win11 home, but using wsl to work in a bash terminal in the windows mount that contains the flowise files. I swapped to 18.17.0 via nvm, reinstalled flowise globally, and ran npx flowise start, then encountered the following error:
npx flowise start
2023-07-27 21:59:30 [INFO]: Starting Flowise...
2023-07-27 21:59:30 [INFO]: ⚡️ [server]: Flowise Server is listening at 3000
2023-07-27 21:59:31 [ERROR]: ❌ [server]: Error during Data Source initialization: SQLITE_READONLY: attempt to write a readonly database
QueryFailedError: SQLITE_READONLY: attempt to write a readonly database
    at Statement.handler (/home/anon/.nvm/versions/node/v18.17.0/lib/node_modules/flowise/node_modules/typeorm/driver/sqlite/SqliteQueryRunner.js:81:26)
    at Statement.replacement (/home/anon/.nvm/versions/node/v18.17.0/lib/node_modules/flowise/node_modules/sqlite3/lib/trace.js:25:27)
    at Statement.replacement (/home/anon/.nvm/versions/node/v18.17.0/lib/node_modules/flowise/node_modules/sqlite3/lib/trace.js:25:27)
    
    Some WARN messages during the flowise installation:
    
npm WARN ERESOLVE overriding peer dependency
npm WARN deprecated @npmcli/move-file@1.1.2: This functionality has been moved to @npmcli/fs
npm WARN deprecated cross-spawn-async@2.2.5: cross-spawn no longer requires a build toolchain, use it instead
npm WARN deprecated vm2@3.9.19: The library contains critical security issues and should not be used for production! The maintenance of the project has been discontinued. Consider migrating your code to isolated-vm.
npm WARN deprecated highlight.js@9.15.10: Version no longer supported. Upgrade to @latest

It will run and I can access the frontend, but any action I take (edit a flow, look at marketplace, etc) causes the application to terminate with no further logs than the error described above.
